src/main/resources/public/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Cook It Best</title>
+    <meta content="width=device-width, initial-scale=1" name="viewport">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
+    <script src="https://unpkg.com/vue@next"></script>
+    <script src="https://unpkg.com/vuex@next"></script>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+    <div id="dishorg">
+        <audio id="success">
+            <source src="http://soundbible.com/grab.php?id=1003&type=mp3" type="audio/mp3">
+            <source src="http://soundbible.com/grab.php?id=1003&type=wav" type="audio/wav">
+        </audio>
+        <audio id="fail">
+            <source src="http://soundbible.com/grab.php?id=1945&type=mp3" type="audio/mp3">
+            <source src="http://soundbible.com/grab.php?id=1945&type=wav" type="audio/wav">
+        </audio>
+        <header>
+            <img id="logo" src="images/logo.svg">
+            <nav>
+                <ul id="nav">
+                    <li><a href="#">Make A Grocery List</a></li>
+                    <li><a href="#">Help</a></li>
+                    <li><a href="#">Plans</a></li>
+                </ul>
+            </nav>
+            <div>
+                <button click="openModal" id="modal-btn"><img
+                        src="https://cdn1.iconfinder.com/data/icons/foxy-universal-circle-glyph/24/circle-round-userinterface-people-account-person-profile-512.png">
```

Comment:
Pull the `<img` down from the previous line for better readability.

Reply:
Done.